src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1253,23 +1285,32 @@ def create_command_arguments(self, command: str) -> str:
             self.rules, convert_path_to_se_path(self.path), command
         )
 
-    def add_on_child_created(self, child_type: str, cb: Callable) -> EventSubscription:
+    def add_on_child_created(
+        self, child_type: str, cb: Callable[[PyMenuT], None]
+    ) -> EventSubscription:
         """Register a callback for when a child object is created.
 
         Parameters
         ----------
         child_type : str
             Type of the child object
-        cb : Callable
+        cb : Callable[[PyMenuT], None]
             Callback function
 
         Returns
         -------
         EventSubscription
             EventSubscription instance which can be used to unregister the callback
         """
+
+        def cb_service(child_path: str):
```

Comment:
In the `add_on_child_created` method in the service class, the callback takes the child path.

I'm not sure if it is possible to pass either object or child path to the service level callback as we have only one implementation of the `add_on_child_created` method of the PyMenu class.